lib/contentful/delivery.ex

```diff
@@ -102,52 +110,57 @@ defmodule Contentful.Delivery do
   end
 
   defp process_url(url) do
-    "#{protocol}://#{endpoint}#{url}"
+    "#{@protocol}://#{@endpoint}#{url}"
   end
 
   defp process_response_body(body) do
     body
-      |> Poison.decode!
-      |> resolve_includes
+    |> Poison.decode!
+    |> resolve_includes
```

Comment:
I will put the `resolve_includes` to `entries` and `entry` then 👍 
However this change is just an indentation change, meaning that it was already included. 
Or, is there something else that I can't figure ?